backend/opre_ops/django_config/settings/local.py

```diff
@@ -5,9 +5,17 @@
 These are settings for local development only, not cloud or production environments.
 """
 # Import all common settings relevant to both local & cloud:
-from opre_ops.django_config.settings.common import *  # noqa: F403
+from opre_ops.django_config.settings.common import *  # noqa: F403, F401
 from opre_ops.django_config.settings.helpers.random_string import generate_random_string
 
+__all__ = [
+    "DATABASES",
+    "SECRET_KEY",
+    "DEBUG",
+    "ALLOWED_HOSTS",
+    "CORS_ALLOWED_ORIGIN_REGEXES",
+    "INSTALLED_APPS",
+]
```

Comment:
Did you mean to keep this in?  We didn't do the same in `common.py`.

Reply:
Yes you can, and I thought about that, but actually liked the idea of having it separate; since we effectively break our code/language by area (python:backend, js:frontend). The time required is the same with minimum overhead on each; so might save 10 seconds on the build/cleanup. I'll play with a few other cleanup steps to and see where it goes; There are a few other break-outs I'd prefer to do across the pipeline as well.